src/standalone/Soil/boundary_conditions.jl

```diff
@@ -259,14 +263,15 @@ function ClimaLSM.boundary_flux(
     p::NamedTuple,
     t,
 )::ClimaCore.Fields.Field
+    FT = eltype(Δz)
```

Comment:
This is totally fine, but Im wondering if it is easier to get FT from the parametric type of the BC or if that cleaner or if there is no preference.

Reply:
I don't think we're storing FT in the BC (code [here](https://github.com/CliMA/ClimaLSM.jl/blob/main/src/standalone/Soil/boundary_conditions.jl#L14)). We could add it as a type parameter for BCs, but I think using dz is easier

Reply:
good point!

Reply:
oh - whatever you think is best is fine - but I just noticed that we have FT available in the model also, and that is an argument.